pkg/catalog/helpers_test.go

```diff
@@ -53,7 +53,7 @@ func newFakeMeshCatalogForRoutes(t *testing.T, testParams testParams) *MeshCatal
 
 	stop := make(chan struct{})
 
-	certManager := tresor.NewFakeCertManager(mockConfigurator)
```

Comment:
so turns out this isn't even used... let's go ahead and remove it from the `MeshCatalog` type altogether, in pkg/catalog/types.go (the certManager field), and all related instances.